inspirehep/modules/records/mappings/records/hep.json

```diff
@@ -211,6 +211,10 @@
                             },
                             "type": "string"
                         },
+                        "full_name_normalized": {
+                            "index": "not_analyzed",
+                            "type": "string"
+                        },
```

Comment:
Why can't this be a field of `full_name`, instead of another key?

Reply:
Ah, because this is not a variant of full_name, is a completely new thing that you are creating in Python...